lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php

```diff
@@ -1983,6 +1984,18 @@ public function mapField(array $mapping) : array
         $this->checkDuplicateMapping($mapping);
         $this->typeRequirementsAreMet($mapping);
 
+        $deprecatedTypes = [
+            Type::BOOLEAN => Type::BOOL,
+            Type::INTEGER => Type::INT,
+        ];
+        if (isset($deprecatedTypes[$mapping['type']])) {
+            @trigger_error(sprintf(
+                '"%s" type was deprecated in 2.1 and will be removed in 3.0. Use "%s" instead.',
```

Comment:
Missed this the first time around: please provide the package name in the message:
```suggestion
                '"%s" type was deprecated in doctrine/mongodb-odm 2.1 and will be removed in 3.0. Use "%s" instead.',
```

Reply:
Added in const as well 👍